toxcore/TCP_connection.c

```diff
@@ -51,6 +51,11 @@ const uint8_t *tcp_connections_public_key(const TCP_Connections *tcp_c)
 }
 
 
+uint32_t tcp_connections_count(const TCP_Connections *tcp_c)
```

Comment:
We could use this in the places where we write `tcp_c->tcp_connections_length` directly in the code?

Reply:
We could. I'll open a separate PR for this.